docs/configuration.md

```diff
@@ -215,6 +216,11 @@ spec:
 | `credentials.appSecret.secretName`                                                                           | SecretName is the name of the secret                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
 | `credentials.token`                                                                                          | This needs to be at least 32 characters a-zA-z It is a preshared key between the node agents and the cluster agent                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
 | `credentials.useSecretBackend`                                                                               | UseSecretBackend use the Agent secret backend feature for retreiving all credentials needed by the different components: Agent, Cluster, Cluster-Checks. If `useSecretBackend: true`, other credential parameters will be ignored. default value is false.                                                                                                                                                                                                                                                                                                                                                                                             |
+| `features.orchestrator_explorer.enabled`                                                                     | Enable this to activate live kubernetes monitoring on the DCA See: https://docs.datadoghq.com/infrastructure/livecontainers/#kubernetes-resources-1. Note: this features the process-agent running. Live containers are enough to be activated See: https://docs.datadoghq.com/infrastructure/process/?tab=linuxwindows. Will add the process agent and auto activate live containers if not set otherwise (disabled)                                                                                                                                                                                                                                  |
```

Comment:
```suggestion
| `features.orchestrator_explorer.enabled`                                                                     | Enable this to activate live Kubernetes monitoring on the DCA, see: https://docs.datadoghq.com/infrastructure/livecontainers/#kubernetes-resources-1. Note: This features the process Agent running. Live containers are enough to be activated, see: https://docs.datadoghq.com/infrastructure/process/?tab=linuxwindows. Adds the process Agent and auto activates live containers if not set otherwise (disabled)                                                                                                                                                                                                                                  |
```